dictionary.json

```diff
@@ -13124,7 +13671,7 @@
     "keywords": [],
     "frame": "a",
     "distribution": "n",
-    "pronominal_class": "fuy",
+    "pronominal_class": "",
     "notes": [],
     "examples": [],
     "fields": []
```

Comment:
Below this: https://toaq.net/refgram/morphology/#mua4 shows that poaq now means "to break sth."